hr_attendance_report_theoretical_time/models/resource_calendar_leaves.py

```diff
@@ -0,0 +1,13 @@
+from odoo import api, models
+
+
+class ResourceCalendarLeaves(models.Model):
+    _inherit = 'resource.calendar.leaves'
+
+    @api.model
+    def search(self, args, offset=0, limit=None, order=None, count=False):
+        new_args = args
```

Comment:
No need to create a new variable. Also add a docstring about why do you overwrite this.